Doc/library/http.cookies.rst

```diff
@@ -137,11 +137,19 @@ Morsel Objects
    * ``secure``
    * ``version``
    * ``httponly``
+   * ``samesite``
 
    The attribute :attr:`httponly` specifies that the cookie is only transferred
    in HTTP requests, and is not accessible through JavaScript. This is intended
    to mitigate some forms of cross-site scripting.
 
+   The attribute :attr:`samesite` specifies that browser is not allowed to send the
+   cookie along with cross-site requests. This help to mitigate CSRF attacks. Valid
+   values for this attribute are "Strict" and "Lax".
+
+   .. versionchanged:: 3.7
```

Comment:
So that should be like this, (description of attribute also moved under 3.7)

```
+   .. versionchanged:: 3.7
+      Added support for :attr:`samesite` attribute.
+
+      The attribute :attr:`samesite` specifies that browser is not allowed to send the
+      cookie along with cross-site requests. This help to mitigate CSRF attacks. Valid
+      values for this attribute are "Strict" and "Lax".
+
```